### PR TITLE
Feat/user service

### PR DESCRIPTION
**Pull request by surya1024**
Implemented user related CURD operations

> UserDAO
> UserService
> UserController 
> @bhavindesai20 review and give your comments

### CODE REVIEW

File: src/com/atlas/entity/User.java [hunk 1]
```diff
@@ -1,14 +1,33 @@
...
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "id")
 	private int id;
-	private String userName;
+
+	@Column(name = "firstName", nullable = false)
+	private String firstName;
+
+	@Column(name = "firstName")
```
Comment (bhavindesai20) [inline src/com/atlas/entity/User.java]: change this to "lastName"

File: src/com/atlas/entity/User.java [hunk 2]
```diff
@@ -1,14 +1,33 @@
...
+	private String LastName;
+
+	@Column(name = "firstName", unique = true, nullable = false)
```
Comment (bhavindesai20) [inline src/com/atlas/entity/User.java]: update this with email

File: src/com/atlas/entity/User.java [hunk 3]
```diff
@@ -1,14 +1,33 @@
...
 	private String email;
+
+	@Column(name = "firstName", nullable = false)
```
Comment (bhavindesai20) [inline src/com/atlas/entity/User.java]: update this with pass word.

**Comment by surya1024** [comment]
Updated @bhavindesai20

**Comment by bhavindesai20** [comment]
:+1:

**Pull request merged**